compiler/src/iree/compiler/InputConversion/StableHLO/LegalizeCHLO.cpp

```diff
@@ -929,9 +929,9 @@ static Value materializeErfApproximationF32ForMagnitudeLeOne(
   assert(x.getType().cast<ShapedType>().getElementType().isF32() &&
          "expect f32 element type");
   const float kErfTCoefficients[] = {
-      +7.853861353153693E-5, -8.010193625184903E-4, +5.188327685732524E-3,
-      -2.685381193529856E-2, +1.128358514861418E-1, -3.761262582423300E-1,
-      +1.128379165726710E+0,
+      +7.853861353153693E-5f, -8.010193625184903E-4f, +5.188327685732524E-3f,
```

Comment:
What's up with these changes?

Reply:
fixing warnings that have crept in (implicit double to float conversion, because these are float arrays but initialized with double values, which is confusing)